line-sdk/src/main/java/com/linecorp/linesdk/internal/nwclient/core/ChannelServiceHttpClient.java

```diff
@@ -229,14 +254,14 @@ private HttpURLConnection openPostConnection(
         conn.setRequestProperty("Content-Length", String.valueOf(postDataSizeByte));
         conn.setConnectTimeout(connectTimeoutMillis);
         conn.setReadTimeout(readTimeoutMillis);
-        conn.setRequestMethod("POST");
+        conn.setRequestMethod(HTTP_METHOD.POST.name());
```

Comment:
We can make the `setRequestMethod` to take the `HttpMethod` directly instead of String.

Reply:
`conn` object is of type `java.net.HttpURLConnection`. In Java implementation, there's no elegant way to change its function from taking String argument to enum type.

Reply:
I didn't notice that the `conn` is `HttpURLConnection`, please ignore this comment.